mlrun/runtimes/nuclio/application/application.py

```diff
@@ -377,6 +382,11 @@ def create_api_gateway(
             ),
         )
 
+        if ssl_redirect or mlrun.mlconf.is_running_on_iguazio():
+            # force ssl redirect so that the application is only accessible via https
+            api_gateway.with_force_ssl_redirect()
```

Comment:
I think this is confusing that the default is false and if in iguazio you set it to true.
If you really want to set the default to false for open source I think it should be in computed in mlrun config and then you can direct the users to check the config instead of looking the the param default value

Reply:
Do you mean having a config value / method of wether to enable ssl redirection?
I do however want to allow setting it explicitly per function, with a flag on `deploy`.